protoc-cli/src/main/java/io/ballerina/protoc/builder/stub/Message.java

```diff
@@ -0,0 +1,231 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Check other places as well

Reply:
Updated in [f61b5f8](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/f61b5f846e6bc8217bd372cde64e595075a417bc)